Unused Imports

```
./src/app/api/controllers/user.py:7:1: F401 'app.utils.get_password_hash' imported but unused
./src/app/repositories/base.py:9:1: F401 'sqlalchemy.orm.sessionmaker' imported but unused
./src/app/repositories/base.py:11:1: F401 'app.db.get_postgres_connection' imported but unused
```

Just  noticed these on my way through, intentional?

--- a/app/api/controllers/user.py
+++ b/app/api/controllers/user.py
@@ -4,7 +4,6 @@ from starlite import Controller, Provide, delete, get, post, put
 from app.constants import USER_CONTROLLER_PATH
 from app.models import UserCreateDTO, UserReadDTO
 from app.repositories import UserRepository
-from app.utils import get_password_hash
 
 
 class UserController(Controller):

--- a/app/repositories/base.py
+++ b/app/repositories/base.py
@@ -6,9 +6,7 @@ from pydantic import BaseModel
 from sqlalchemy import select
 from sqlalchemy.exc import SQLAlchemyError
 from sqlalchemy.ext.asyncio import AsyncSession
-from sqlalchemy.orm import sessionmaker
 
-from app.db import get_postgres_connection
 from app.exceptions import RepositoryException
 from app.models.base import Base
 

--- a/app/repositories/user.py
+++ b/app/repositories/user.py
@@ -1,6 +1,5 @@
 from typing import cast
 
-from pydantic import BaseModel
 from sqlalchemy import select
 from sqlalchemy.exc import SQLAlchemyError
 

